documentation/docu.adoc

```diff
@@ -0,0 +1,30 @@
+= Strimzi: Kafka as a Service running on Kubernetes and OpenShift
+Version 0.1
+:sectnums:
+:toc: preamble
+:toclevels: 3
+:toc-title: Content
+:imagesdir: ./img
+:xrefstyle: full
+
+This guide describes how to install and use Strimzi. Strimzi provides a way to run an Apache Kafka cluster on Kubernetes
+or OpenShift in various deployment configurations.
+
+== Overview
+
+Apache Kafka is a popular platform for streaming data delivery and processing. For more details about Apache Kafka
+itself, visit http://kafka.apache.org[Apache Kafka website]. Strimzi is a project which aims to make it easier to run
+Apache Kafka on top of Kubernetes and OpenShift.
```

Comment:
I think this phrasing is better: "The aim of the Strimzi project is to make it easy to run Apache Kafka on Kubernetes and OpenShift".

Not "easier", as that suggests it's somewhat easy to start with (and it's not!)